tests/foreman/installer/test_infoblox.py

```diff
@@ -15,183 +17,162 @@
-
-    :id: ba957e82-79bb-11e6-94c5-68f72889dc7f
-
-    :Steps: Provision a host with infoblox as dhcp provider
-
-    :expectedresults: Check host ip is on infoblox range configured by
-        option --foreman-proxy-plugin-dhcp-infoblox-use-ranges=true
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-@pytest.mark.upgrade
-def test_dns_records():
-    """Check DNS records are updated via infoblox DNS plugin
-
-    :id: 007ad06e-79bc-11e6-885f-68f72889dc7f
-
-    :Steps:
-
-        1. Provision a host with infoblox as dns provider
-        2. Update a DNS record on infoblox
-
-    :expectedresults: Check host dns is updated accordingly to infoblox
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
+import requests
+from fauxfactory import gen_mac
+from fauxfactory import gen_string
+from requests.exceptions import HTTPError
+
+from robottelo.config import settings
+from robottelo.utils.installer import InstallerCommand
+
+
+infoblox_plugin_enable = [
+    'enable-foreman-proxy-plugin-dhcp-infoblox',
+    'enable-foreman-proxy-plugin-dns-infoblox',
+]
+infoblox_plugin_opts = {
+    'foreman-proxy-dhcp': 'true',
+    'foreman-proxy-dhcp-managed': 'false',
+    'foreman-proxy-dhcp-provider': 'infoblox',
+    'foreman-proxy-dhcp-server': settings.infoblox.hostname,
+    'foreman-proxy-plugin-dhcp-infoblox-dns-view': 'default',
+    'foreman-proxy-plugin-dhcp-infoblox-network-view': 'default',
+    'foreman-proxy-plugin-dhcp-infoblox-username': settings.infoblox.username,
+    'foreman-proxy-plugin-dhcp-infoblox-password': settings.infoblox.password,
+    'foreman-proxy-plugin-dhcp-infoblox-record-type': 'fixedaddress',
+    'foreman-proxy-dns': 'true',
+    'foreman-proxy-dns-provider': 'infoblox',
+    'foreman-proxy-plugin-dns-infoblox-username': settings.infoblox.username,
+    'foreman-proxy-plugin-dns-infoblox-password': settings.infoblox.password,
+    'foreman-proxy-plugin-dns-infoblox-dns-server': settings.infoblox.hostname,
+    'foreman-proxy-plugin-dns-infoblox-dns-view': 'default',
+}
+
+
+@pytest.mark.stream
+@pytest.mark.e2e
+@pytest.mark.parametrize('module_sync_kickstart_content', [8], indirect=True)
+def test_infoblox_end_to_end(
+    request,
+    module_sync_kickstart_content,
+    module_provisioning_capsule,
+    module_target_sat,
+    module_sca_manifest_org,
+    module_location,
+    module_default_org_view,
+    module_lce_library,
+    default_architecture,
+    default_partitiontable,
+):
+    """Verify end to end infoblox plugin integration and host creation with
+       Infoblox as DHCP and DNS provider
+
+    :id: 0aebdf39-84ba-4182-9a17-6eb523f1695f
+
+    :steps:
+        1. Run installer to integrate Satellite and Infoblox.
+        2. Assert if DHCP, DNS provider and server is properly set.
+        3. Create a host with proper domain and subnet.
+        4. Check if A record is created with host ip and hostname on Infoblox
```

Comment:
: )
```suggestion
        4. Check if a record is created with host ip and hostname on Infoblox
```

Reply:
It is correct :)

Reply:
I know this is resolved, but just for a bit of additional context, an "A Record" refers specifically to an IPv4 address record. An IPv6 address record is referred to as an "AAAA Record," which is usually pronounced "quad A record" when spoken aloud.